examples/stable-diffusion/README.md

```diff
@@ -294,3 +294,58 @@ image = pipe(prompt, num_inference_steps=50, guidance_scale=7.5).images[0]
 
 image.save("cat-backpack.png")
 ```
+
+
+## Fine-Tuning
+
+The `train_text_to_image*.py` scripts shows how to implement the fine-tuning of Stable Diffusion models on Habana Gaudi.
+
+### Example for SD
+We can launch the fine-tuning of SD model using:
+
+#!/bin/bash
+python train_text_to_image.py \
+  --pretrained_model_name_or_path CompVis/stable-diffusion-v1-4 \
+  --dataset_name lambdalabs/pokemon-blip-captions \
+  --resolution=512 \
+  --center_crop \
+  --random_flip \
+  --train_batch_size=1 \
+  --gradient_accumulation_steps 4 \
+  --gradient_checkpointing \
+  --max_train_steps 50 \
+  --learning_rate 1e-05 \
+  --max_grad_norm 1 \
+  --lr_scheduler constant \
+  --lr_warmup_steps 0 \
+  --output_dir sd-pokemon-model \
+  --gaudi_config_name Habana/stable-diffusion \
+  --use_hpu_graphs \
+  --throughput_warmup_steps 3 \
+  --bf16
+
```

Comment:
suggest to remove sd related, and you can submit later with just new_version of train_tex_to_image.py

Reply:
Removed as suggested